packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -82,6 +82,49 @@ export const bm25s = (model: ModelData): string[] => [
 retriever = BM25HF.load_from_hub("${model.id}")`,
 ];
 
+export const depth_anything_v2 = (model: ModelData): string[] => {
+	let encoder: string;
+	let features: number;
+	let out_channels: string;
+
+	encoder = "vits";
+	features = 64;
+	out_channels = "[48, 96, 192, 384]";
```

Comment:
For unknown model ids I would still explicitly set a placeholder as mentioned in https://github.com/huggingface/huggingface.js/pull/785#discussion_r1677327853 (rather than default values that looks like real values). Otherwise PR looks good to me :)

Reply:
Ok I see, made them all string now